### PR TITLE
 OE-764: Fix navigation function in playwright (clever login failure)

### PR DESCRIPTION
**Pull request by ayan1229**

### CODE REVIEW

File: apps/oew/playwright/features/cleverSignIn.feature [hunk 1]
```diff
@@ -10,9 +10,6 @@ Feature: As a user I want to sign into Open Ebooks Web by using clever credentia
      And I fill in the "clever username textfield" with "OPENEBOOKS_CLEVER_HIGH_SCHOOL_USER_USERNAME"
      And I fill in the "clever password textfield" with "OPENEBOOKS_CLEVER_HIGH_SCHOOL_USER_PW"
      And I click the "clever login button"
-     And I am directed to the "sign in clever callback" page
-     And I am directed to the "open ebook circulation" page
      And I click the "sign out button"
```
Comment (samanthaandrews) [inline apps/oew/playwright/features/cleverSignIn.feature:13]: Let's add a step here to confirm that after logging in with Clever, they see the Circulation landing page

**Review by samanthaandrews — CHANGES_REQUESTED** [review]
Just one small change, otherwise this looks good. You'll also need to fix merge conflicts in the CHANGELOG.md file.

**Review by samanthaandrews — APPROVED** [review]

**Comment by kristojorg** [comment]
Is this ready to merge?

**Comment by samanthaandrews** [comment]
yeah it’s ready

**Pull request merged**